introjs/build.boot

```diff
@@ -0,0 +1,29 @@
+(set-env!
+ :resource-paths #{"resources"}
+ :dependencies '[[cljsjs/boot-cljsjs "0.5.2" :scope "test"]])
+
+(require '[cljsjs.boot-cljsjs.packaging :refer :all])
+
+(def +lib-version+ "2.4.0")
+(def +version+ (str +lib-version+ "-0"))
+
+(task-options!
+ pom {:project     'cljsjs/introjs
+      :version     +version+
+      :description "Step-by-step guide and feature introduction"
+      :url         "http://introjs.com/"
+      :scm         {:url "https://github.com/usablica/intro.js"}
+      :license     {"GNU AGPLv3" "https://www.gnu.org/licenses/agpl-3.0.en.html"}})
+
+(deftask package []
+  (comp
+   (download :url (str "https://github.com/usablica/intro.js/archive/v" +lib-version+ ".zip")
+             :unzip true)
+   (sift :move {#"intro.js-2.4.0/intro.js"                 "cljsjs/introjs/development/introjs.inc.js"
```

Comment:
Please write these regexes in a way the don't need to be changed when the version is updated,

e.g. #"intro.js-[^/]*/intro.js" should accept anything but forward slashes in place of of the version number.